internal/runners/use/reset.go

```diff
@@ -0,0 +1,48 @@
+package use
+
+import (
+	"runtime"
+
+	"github.com/ActiveState/cli/internal/config"
+	"github.com/ActiveState/cli/internal/globaldefault"
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/output"
+	"github.com/ActiveState/cli/internal/subshell"
+)
+
+type Reset struct {
+	out      output.Outputer
+	config   *config.Instance
+	subshell subshell.SubShell
+}
+
+func NewReset(prime primeable) *Reset {
+	return &Reset{
+		prime.Output(),
+		prime.Config(),
+		prime.Subshell(),
+	}
+}
+
+func (u *Reset) Run() error {
+	logging.Debug("Resetting default project runtime")
+
+	reset, err := globaldefault.ResetDefaultActivation(u.subshell, u.config)
```

Comment:
Do we want a prompt here to confirm the reset? @Naatan @rawktron

Reply:
I went ahead and added the prompt, like we have for similar reset-type of commands.